app/mailers/user_email_mailer.rb

```diff
@@ -18,4 +18,23 @@ def response_invite(user, time_period)
     @table = @table.transpose.flatten
     mail(to: user.email, subject: "Hey #{user.first_name}, how was your week?")
   end
+
+  def results_email(user, time_period, words)
+    # //Will have to check back in the future when Epic: Results is done
+    general_link = api_v1_see_the_results_url(time_period_id: time_period.id, user_id: user.id, not_working: false)
+    uri = URI.parse(general_link)
+    query_params = URI.decode_www_form(uri.query || '') << ['last_step', 'results']
+    uri.query = URI.encode_www_form(query_params)
+    @link_see_the_results = uri.to_s
+    # Will have to check back in the future when Epic: Results is done\\
+    @view_calendar_days = "#{time_period.start_date.strftime('%d %b.')} \n – \n#{time_period.end_date.strftime('%d %b.')}"
+    @user = user
+    @time_period = time_period
+    @words = words
+    mail(to: @user.email, subject: "Hey #{user.first_name}, the results are in!")
+  end
+
+  def range_format(date)
+    "#{date.start_date.strftime('%d')}-#{date.end_date.strftime('%d')} #{date.end_date.strftime('%b')}"
+  end
```

Comment:
@LyubovP could you explain please where you use this method?

Reply:
@SBorozenets The range_format method was left over from when we used to format the date for the @view_calendar_days variable. At the moment, this method is indeed not being used in our code. I have removed it. Thank you for pointing that out.